tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -268,5 +268,257 @@ describe('Test Fleet access with RBAC with custom roles using Standard User', {
+  })
+
+
+  qase(46,
+    it('Fleet-46: Test "Standard-user" | Custom Role | Fleetworkspaces, Bundles = [ALL] | GitRepos = [List]', { tags: '@fleet-46' }, () => {
+      
+      const stduser = "std-user-46"
+      
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName_1)
+      
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // CAN go to Continuous Delivery Dashboard and "list" gitrepos
+      cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
+      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
+      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');
+      
+      // CHECKS IN  FLEET-DEFAULT
+      // Can't "Create", "Edit" nor "Delete"
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.get('.btn.role-primary').contains('Add Repository').should('not.exist');
+      // Note: listing is checked implictly here
+      cy.open3dotsMenu(repoNameDefault, 'Edit Config', true);
+      cy.open3dotsMenu(repoNameDefault, 'Delete', true);
+      // CHECKS IN  FLEET-DEFAULT
+      // Can't "Create", "Edit" nor "Delete"
+      cy.fleetNamespaceToggle('fleet-local');
+      cy.open3dotsMenu(repoName, 'Edit Config', true);
+      cy.open3dotsMenu(repoName, 'Delete', true);
+    })
+  )
+
+  qase(47,
+    it('Fleet-47: Test "Standard-user" | Custom Role | Fleetworkspaces, Bundles = [ALL] | GitRepos = [List, Create]', { tags: '@fleet-47' }, () => {
+      
+      const stduser = "std-user-47"     
+      
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName_2)
+      
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // CAN go to Continuous Delivery Dashboard and "list" gitrepos
+      cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
+      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
+      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');
+
+      // CHECKS IN  FLEET-DEFAULT
+      // CAN "Create" repos
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.clickButton('Add Repository');
+      cy.contains('Git Repo:').should('be.visible');
+      // Can't "Edit" nor "Delete" repos
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.open3dotsMenu(repoNameDefault, 'Edit Config', true);
+      cy.open3dotsMenu(repoNameDefault, 'Delete', true);
+      
+      // CHECKS IN  FLEET-LOCAL
```

Comment:
```suggestion
      // CHECKS IN FLEET-LOCAL
```

Reply:
Thanks, addressed in `3f474bf958123b38439a2f4d1683ac3dcf0b3f16`